app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-form-tracker.js

```diff
@@ -84,12 +84,13 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       var labelText = input.label.innerText || input.label.textContent
       var inputType = elem.getAttribute('type')
       var inputNodename = elem.nodeName
+      var inputTypes = ['text', 'search', 'email', 'number']
 
       if (inputType === 'checkbox' && elem.checked) {
         input.answer = labelText
       } else if (inputNodename === 'SELECT' && elem.options[elem.selectedIndex].value) {
         input.answer = elem.options[elem.selectedIndex].text
-      } else if ((inputType === 'text' || inputType === 'search') && elem.value) {
+      } else if (inputTypes.includes(inputType) && elem.value) {
```

Comment:
Quick question - instead of `inputTypes.includes(inputType)`, would we want to use `inputTypes.indexOf(inputType) !== -1`? Only because I've seen it elsewhere in our GA4 code and I guess would be beneficial if we want to gather data from any remaining IE11 users.

Reply:
Great shout, thanks @JamesCGDS, have updated 👍